.NET/Microsoft.Recognizers.Text/Models/ResolutionKey.cs

```diff
@@ -11,5 +11,7 @@ public static class ResolutionKey
         public const string SubType = "subtype";
         public const string Offset = "offset";
         public const string RelativeTo = "relativeTo";
+        public const string Issuer = "issuer";
```

Comment:
This is not ideal, but can be acceptable.
It would be better to do how DateTime does it.